File: DWR neu/DWR/DWR/Model/Rohstoffe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DWR.Model
{
    public class Rohstoffe
    {
        public int holz { get; set; }
        public int stein { get; set; }
        public int lehm { get; set; }

        public Rohstoffe(int _holz, int _stein, int _lehm) {
            this.holz = _holz;
            this.stein = _stein;
            this.lehm = _lehm;
        }

        public Rohstoffe() { }
    }
}

File: DWR Abgabe Ostern 18.03.2016/DWR_WebServer/src/pkgModel/Rohstoffe.java
package pkgModel;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement (name = "DWR")
public class Rohstoffe {
	private int holz;
	private int stein;
	private int lehm;
	
	public Rohstoffe() {
		super();
		this.holz = 0;
		this.stein = 0;
		this.lehm = 0;
	}
	
	public Rohstoffe(int holz, int stein, int lehm) {
		super();
		this.holz = holz;
		this.stein = stein;
		this.lehm = lehm;
	}
	@Override
	public String toString() {
		return "Rohstoffe [holz=" + holz + ", stein=" + stein + ", lehm="
				+ lehm + "]";
	}
	public int getHolz() {
		return holz;
	}
	public void setHolz(int holz) {
		this.holz = holz;
	}
	public int getStein() {
		return stein;
	}
	public void setStein(int stein) {
		this.stein = stein;
	}
	public int getLehm() {
		return lehm;
	}
	public void setLehm(int lehm) {
		this.lehm = lehm;
	}
}
